graphs/possible_bipartition.py

```diff
@@ -1,12 +1,32 @@
-# Can be used for BFS
 from collections import deque 
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(NE)
+        Space Complexity: O(N)
```

Comment:
👍   I do think however that the time complexity is O(N + E) because you hit every node and check each edge that node has, but you never repeat a node.